game.py

```diff
@@ -106,6 +101,10 @@ def read_file(self, file_path):
         pass
 
     def run(self):
+        self.brick_hit_sound  = pygame.mixer.Sound("assets/brickhit.ogg")
+        self.paddle_hit_sound = pygame.mixer.Sound("assets/paddlehit.ogg")
+        self.brick_hit_sound.set_volume(0.8)
+        
```

Comment:
Why move these here?

Reply:
In activity.py, a new object of BallAndBrick class is created before the call to PygameCanvas constructor (where all the modules are initialized). So the code inside \_\_init__() would run before any modules are initialized resulting in `mixer not initialized` error. Moving these lines to the run function solves this problem.

Reply:
I agree, do not call pygame in `__init__`, call in `run`.